### PR TITLE
Doc changes per isis review

### PR DESCRIPTION
**Pull request by CatFreed**
Making minor doc changes that Gerard has suggested

### CODE REVIEW

File: docs/working-with-the-iobc.rst [hunk 1]
```diff
@@ -754,6 +754,8 @@ SPI
 The external SPI bus is not currently available to the userspace. It
 will be added in a future release.
 
+.. _user-accounts:
+
 User Accounts
```
Comment (jacoffey3) [inline docs/working-with-the-iobc.rst]: When should they create a user account? Is that a necessary step (necessary as in we want them to do it?)

Comment (CatFreed) [inline docs/working-with-the-iobc.rst]: It's not a necessary step. We include the kubos/Kubos123 user account for this purpose.

A recommended step would be changing the root password, though...

**Comment by CatFreed** [comment]
[ISIS Review - Doc Fixes](https://trello.com/c/Awm7YA4A/27-isis-review-doc-fixes)

File: docs/working-with-the-iobc.rst (hunk 1 above)
Comment (jacoffey3) [inline docs/working-with-the-iobc.rst]: Whoops. Forgot to check back in. Sorry.

**Pull request merged**